algorithm.py

```diff
@@ -52,7 +60,7 @@ def get_graph_from_full_adj_mat(full_adjacency_matrix: np.ndarray):
         edge_count = full_adjacency_matrix.shape[1]
         g.add_vertices(vertex_count)
         edges = np.empty((edge_count, 2), dtype=int)
-        directed_flags = np.empty((edge_count, 1), dtype=bool)
+        directed_flags = np.empty(edge_count, dtype=bool)
```

Comment:
czy teraz przypadkiem nie robisz macierzy kwadratowej?

Reply:
Z tego co pamiętam to wtedy się błednie robiło przypisanie `g.es['directed'] = directed_flags.tolist()`

Reply:
Mój błąd.